### PR TITLE
Fix #28 and pass bundled files to emiter

### LINKED ISSUE #28: rollem --watch only builds sources after the first change (incompatibility with rollup --watch)
Hi!

Please take a look at the following files:

**package.json**

```javascript
{
  "name": "issue",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "rollem": "rollem -c rollem.config.js",
    "rollemWatch": "npm run rollem -- --watch",
    "rollup": "rollup -c rollup.config.js",
    "rollupWatch": "npm run rollup -- -w"
  },
  "author": "",
  "license": "ISC",
  "dependencies": {
    "rollem": "^1.10.0",
    "rollup": "^0.41.4",
    "rollup-watch": "^3.2.2"
  }
}
```

**rollup.config.js**

```javascript
export default {
  entry: 'src/index.js',
  dest: 'dist/script.js'
};
```

**rollem.config.js**

```javascript
export default [{
  entry: 'src/index.js',
  dest: 'dist/script.js'
}];
```

Finally create an `src` folder with an `index.js` in it with any content.

If you run `npm run rollupWatch`, then the bundle gets built first and gets rebuilt after every change.
Opposed to this when you run `npm run rollemWatch`, there is no building of the bundle, it only gets built once you do any changes in the source file.

### PR DESCRIPTION
**Pull request by notetiene**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -61,13 +61,14 @@ function rollem (configs, options) {
       if (err) {
         watcher.emit('failed', err)
       } else {
-        gazeWatcher.on('all', () => {
-          watcher.emit('changed')
-          buildBundles(configs).then(() => watcher.emit('rolled'))
+        gazeWatcher.on('all', (_event, filepath) => {
```
Comment (meszaros-lajos-gyorgy) [inline src/index.js:64]: Just out of curiosity: why do you need the underscore character for the event variable? Naming it simply "event" seems fine for me.

**Review by meszaros-lajos-gyorgy — APPROVED** [review]
Nice job debugging #28

File: src/index.js (hunk 1 above)
Comment (notetiene) [reply src/index.js:64]: It's a common notation for unused function parameters. ESLint allow [using a pattern](http://eslint.org/docs/rules/no-unused-vars#argsignorepattern) to ignore those. When configured correctly, ESLint doesn't complain about it.

Comment (notetiene) [reply src/index.js:64]: I think we should add an ESLint configuration file. I'm used adding trailing semi-colons for each statements, but it doesn't seem to be so in that project. So every times I use my test-editor, it complains about missing semi-colons.

Comment (meszaros-lajos-gyorgy) [reply src/index.js:64]: Actually we do have a linter in the project: https://standardjs.com/index.html

Comment (meszaros-lajos-gyorgy) [reply src/index.js:64]: I checked your link and tested out this example code on the demo page, which is similar to your pattern:
```javascript
/* global document */

document.addEventListener('click', function(e, filepath){
  document.body.textContent = filepath
})
```
ESLint responded with everything being okay.

Comment (meszaros-lajos-gyorgy) [reply src/index.js:64]: The rule, which you mentioned checks arguments from right to left. You cannot give parameters to a function and leave out something from the left, but you can ignore things from the right. The rule only tries to narrow down the second case:

```javascript
const fn = (a, b, c) => {
  console.log('I only care about b and c, please don\'t define a', b, c);
};

fn('you cannot skip defining me', 12, 34)
```

( [Maybe python allows something like this](http://stackoverflow.com/q/21694536/1806628), but JS doesn't )

Comment (notetiene) [reply src/index.js:64]: It's probably just my own configuration. I got ESLint activated by default in my text-editor.

Comment (meszaros-lajos-gyorgy) [reply src/index.js:64]: If you want to check the package, you can do it with `npm run lint`, if that says your code is okay, then it should be fine for us too.

**Pull request merged**